core/codec/cbor/cbor_decode_stream.cpp

```diff
@@ -17,15 +17,11 @@ namespace fc::codec::cbor {
     value_.remaining = UINT32_MAX;
   }
 
-  CborDecodeStream &CborDecodeStream::operator>>(std::vector<uint8_t> &bytes) {
-    if (!cbor_value_is_byte_string(&value_)) {
-      outcome::raise(CborDecodeError::WRONG_TYPE);
-    }
-    size_t size;
-    if (CborNoError != cbor_value_get_string_length(&value_, &size)) {
-      outcome::raise(CborDecodeError::INVALID_CBOR);
+  CborDecodeStream &CborDecodeStream::operator>>(gsl::span<uint8_t> bytes) {
+    auto size = bytesLength();
+    if (static_cast<size_t>(bytes.size()) != size) {
```

Comment:
is it ok that I should somehow know the size of vector I'm going to decode?

Reply:
It possibly can broke down current functionality.

Reply:
There is separate function for vector